packages/recipe/src/components/EzProgress/Documentation/Stories/Default.stories.tsx

```diff
@@ -11,5 +70,19 @@ export default meta;
 type Story = StoryObj<typeof EzProgress>;
 
 export const Default: Story = {
-  render: () => <div>Coming soon...</div>,
+  args: {},
```

Comment:
I would recommend adding some args here, otherwise the preview will show an empty progress.

```suggestion
  args: {
    goal: 90,
    label: 'Q1 on-time delivery goal progress - 91%',
    subgoal: 49,
    value: 91,
  },
```